src/Symfony/Component/Amqp/UrlParser.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Symfony\Component\Amqp;
+
+class UrlParser
+{
+    private function __construct()
+    {
+    }
```

Comment:
Maybe add a comment to explain why? It's certainly going to rise some eyebrows...